install_debian_autopsy_4.19.2.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+sudo apt install wget
+
+#install dependencies
+wget https://github.com/sleuthkit/sleuthkit/releases/download/sleuthkit-4.11.1/sleuthkit-java_4.11.1-1_amd64.deb
+wget https://github.com/labcif/autopsy-packager/releases/download/4.19.2-1/autopsy_4.19.2-1_amd64.deb
+
+wget -q -O - https://download.bell-sw.com/pki/GPG-KEY-bellsoft | sudo apt-key add -
+echo "deb [arch=amd64] https://apt.bell-sw.com/ stable main" | sudo tee /etc/apt/sources.list.d/bellsoft.list
```

Comment:
Do we need to print this to stdout?
```suggestion
echo "deb [arch=amd64] https://apt.bell-sw.com/ stable main" > /etc/apt/sources.list.d/bellsoft.list
```